decision_desk/DayOfModel.R

```diff
@@ -0,0 +1,159 @@
+library(tidyverse)
+library(glue)
+library(ggplot2)
```

Comment:
You don't seem to use ggplot. If you run this in the cloud this'll be an extra import that we have to install into the environment. are you sure we need it?

Reply:
You're 100% right, I'll do that right now